src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,70 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final int OPERATION_NAME = 0;
+    private static final int OPERATION_AMOUNT = 1;
+    private static final String REGEX = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String infoAboutBuyOperation = getSumOfOperation(readTheFile(fromFileName), Operation.BUY);
+        String infoAboutSupplyOperation = getSumOfOperation(readTheFile(fromFileName),
+                Operation.SUPPLY);
+        String infoAboutResult = getResult(infoAboutBuyOperation, infoAboutSupplyOperation);
+        writeStatistic(infoAboutBuyOperation, infoAboutSupplyOperation,
+                infoAboutResult, toFileName);
+    }
+
+    private String [] readTheFile(String fromFileName) {
+        File file = new File(fromFileName);
+        List<String> strings;
+        try {
+            strings = Files.readAllLines(file.toPath());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file", e);
+        }
+        String[] readingData = new String[strings.size()];
+        for (int i = 0; i < readingData.length; i++) {
+            readingData[i] = strings.get(i);
+        }
```

Comment:
why do you need to do this conversion if you can return List and work with it? If you don't know how to work with it ok, then overwrite your logic so you won't need to use it at all